Documentation/CRDs/specification.md

```diff
@@ -11852,6 +11852,9 @@ RadosNamespaceMirroringMode
 <tbody><tr><td><p>&#34;image&#34;</p></td>
 <td><p>RadosNamespaceMirroringModeImage represents the image mode</p>
 </td>
+</tr><tr><td><p>&#34;init-only&#34;</p></td>
+<td><p>RadosNamespaceMirroringModeInitOnly represents the image mode</p>
+</td>
```

Comment:
radosnamespace will not have the init-only mode

Reply:
removed